drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/pom.xml

```diff
@@ -30,8 +28,6 @@
       <scope>provided</scope>
     </dependency>
 
-    <!-- MANSTIS - EVERYTHING BELOW THIS HAS BEEN _SANITISED_! -->
```

Comment:
@manstis 
I think @yesamer is trying to cancel one of your best contribution ever! :)